content/02.main-text.md

```diff
@@ -170,7 +183,7 @@ Conferences and open source projects have used codes of conduct to establish the
 We would encourage the maintainers of similar projects to consider broader codes of conduct for project participants that establish on social as well as academic norms.
 
 Open writing presents new opportunities for scholarly communication.
-`TODO: reference "paper of the future"? arXiv:1601.02927 doi:10.22541/au.149693987.70506124 doi:10.22541/au.148769949.92783646 http://blogs.nature.com/naturejobs/2017/06/01/techblog-c-titus-brown-predicting-the-paper-of-the-future`
```

Comment:
Did you intentionally remove this other TechBlog article on the paper of the future?

Reply:
My bad, this change should be reverted. I didn't realize this wasn't a cite for the recent techblog on deep review. Good example of diffs help catch stupid errors!

Reply:
#meta